metadata-ingestion/src/datahub/ingestion/source/tableau_common.py

```diff
@@ -502,7 +502,12 @@ def get_tags_from_params(params: List[str] = []) -> GlobalTagsClass:
 
 
 def tableau_field_to_schema_field(field, ingest_tags):
-    nativeDataType = field.get("dataType", "UNKNOWN")
+    # The check here makes sure that even if 'dataType' key exists in the 'field' dictionary but has value None,
+    # it will be set as "UNKNOWN" (nativeDataType field can not be None in the SchemaField).
+    # Hence, field.get("dataType", "UNKNOWN") is not enough
+    nativeDataType = field.get("dataType")
```

Comment:
```suggestion
    nativeDataType = field.get("dataType") or "UNKNOWN"
```

Reply:
I have considered that solution but `or` will also trigger if `field.get("dataType")` returns some actual value such as `False` or, worse, if it is an empty `string`. If you feel this cases should also be handled as "UNKNOWN" (which I believe so as well, made this build just to test a theory) I will gladly adjust to your change. WDYT?

Reply:
either is probably fine - I expect `field.get("dataType")` is only going to be string or None

I just like the `or` thing because it's more concise, but it's not a particularly important suggestion. we should be fine merging this as long as you've validated that it does fix the bug